keylime/ca_util.py

```diff
@@ -63,15 +63,22 @@ def load_cert_by_path(cert_path: str) -> Certificate:
     return cert
 
 
-def setpassword(pw: Optional[str]) -> None:
+def setpassword(key_store_pw: Optional[str]) -> None:
     global global_password
-    if not pw:
-        pw = getpass.getpass("Please enter the password to decrypt your keystore: ")
+    if not key_store_pw:
+        key_store_pw = config.get("ca", "password")
+        if key_store_pw:
```

Comment:
Should be noted that the default fallback is the empty string, so a missing password will signaled as the empty string.

Reply:
True, wil fix it.